IDE/ECLIPSE/DEOS/README.md

```diff
@@ -166,45 +181,62 @@ userServiceThread              # Server thread template name
 /
 ```
 
-   #### Building and Running
- 1.  Build your project, then load and run your image on a target platform. Review the test results on the console output.
+#### Building and Running
+
+1.  Build your project, then load and run your image on a target
+    platform. Review the test results on the console output.
 
 
 ### `wolfcrypt_test()`
-wolfcrypt_test() prints a message on the target console similar to the following output:
+
+`wolfcrypt_test()` prints a message on similar to the following:
+
 ```
 error    test passed!
 base64   test passed!
 asn      test passed!
 ...
 ```
+
 This example doesn't show the whole output.
 
 ### `benchmark_test()`
-benchmark_test() prints a message on the target console similar to the following output.
+
+`benchmark_test()` prints a message on the similar to the following:
 
 ```
 ------------------------------------------------------------------------------
- wolfSSL version 3.15.5
+wolfSSL version 4.6.0
 ------------------------------------------------------------------------------
 wolfCrypt Benchmark (block bytes 1024, min 1.0 sec each)
-RNG               225 KB tooks 1.026 seconds,  219.313 KB/s
-AES-128-CBC-enc    250 KB toks 1.105 seconds  226.210 KB/s
-AES-128-CBC-dec    225 KB tooks 1.005 seconds,  223.922 KB/s
+RNG                  2 MB took 1.000 seconds,    2.124 MB/s
+AES-128-CBC-enc      5 MB took 1.000 seconds,    5.127 MB/s
+AES-128-CBC-dec      5 MB took 1.000 seconds,    4.907 MB/s
+AES-192-CBC-enc      5 MB took 1.000 seconds,    4.736 MB/s
+AES-192-CBC-dec      5 MB took 1.000 seconds,    4.761 MB/s
```

Comment:
Consider adding the complete benchmark results.

Reply:
I think it varies too much on the platform. Would prefer not to include as a reference or baseline.